internal/scti/signatures_test.go

```diff
@@ -401,3 +402,56 @@ func setupSigner(fakeSig []byte) (crypto.Signer, error) {
 
 	return testdata.NewSignerWithFixedSig(key, fakeSig), nil
 }
+
+func TestBuildCp(t *testing.T) {
+	// Create a test signer.
+	ecdsaSigner, err := loadPEMPrivateKey("../testdata/test_ct_server_ecdsa_private_key.pem")
+	if err != nil {
+		t.Fatalf("Can't open key: %v", err)
+	}
+
+	// Define test data.
+	size := uint64(12345)
+	hash := []byte("test_hash_value_12345678901234567890")
+
+	// Build the checkpoint which is in the RFC6962NoteSignature format.
+	checkpoint, err := buildCp(ecdsaSigner, size, fixedTimeMillis, hash)
+	if err != nil {
+		t.Errorf("buildCp failed: %v", err)
+	}
+
+	// Verify whether the checkpoint is empty.
+	if len(checkpoint) == 0 {
+		t.Errorf("buildCp returned an empty checkpoint")
+	}
+
+	// Verify that the checkpoint can be parsed.
+	var sig RFC6962NoteSignature
+	_, err = tls.Unmarshal(checkpoint, &sig)
+	if err != nil {
+		t.Errorf("failed to unmarshal checkpoint: %v", err)
+	}
+	// Verify the timestamp in the note signature.
+	if sig.Timestamp != fixedTimeMillis {
+		t.Errorf("buildCp returned wrong timestamp, got %d, want %d", sig.Timestamp, fixedTimeMillis)
+	}
+
+	// Verify the signature using the public key.
+	sth := types.SignedTreeHead{
```

Comment:
I've  split +} renamed the packages to rfc6962 and staticct

Reply:
Done.